modules/caddyhttp/reverseproxy/selectionpolicies.go

```diff
@@ -770,7 +770,7 @@ func leastRequests(upstreams []*Upstream) *Upstream {
 		return nil
 	}
 	var best []*Upstream
-	var bestReqs int = -1
+	bestReqs := -1
```

Comment:
Nitpick but this is actually less symmetrical so it looks less nice. But whatever, it's fine.